### PR TITLE
feat: antd error info

### PR DESCRIPTION
**Pull request by BoYanZh**

### CODE REVIEW

File: example/app.py [hunk 1]
```diff
@@ -1,23 +1,78 @@
...
 logging.intercept_all_loggers()
 config.init_settings(FastAPIConfig)
 app = FastAPI()
-app.logger = logger
 
 
-# @errors.register_error_code_model
-# class ErrorCode(errors.ErrorCode):
-#     UserError = "UserError"
+# @register_error_code_model
```
Comment (tc-imba) [inline example/app.py:25]: why register_error_code_model is commented here?

File: fastapi_rest_framework/errors.py [hunk 2]
```diff
@@ -30,28 +30,45 @@ def _generate_enum(cls: Type[ErrorCode]) -> Enum:
 )
 
 
-def register_error_code_model(cls: Type[ErrorCode]) -> Enum:
-    if issubclass(cls, ErrorCode):
-        enum_cls = _generate_enum(cls)
-        _error_code_model.set(enum_cls)
-        return enum_cls
-    raise TypeError(f"{cls} is not subclass of ErrorCode")
+# TODO: fix code completion
```
Comment (tc-imba) [inline fastapi_rest_framework/errors.py:33]: all type hints correct here?

File: fastapi_rest_framework/errors.py [hunk 3]
```diff
@@ -30,28 +30,45 @@ def _generate_enum(cls: Type[ErrorCode]) -> Enum:
...
 
 class BizError(Exception):
-    def __init__(self, error_code: ErrorCode, error_msg: str = ""):
+    # FIXME: schema & type
+    def __init__(self, error_code: Union[str, ErrorCode], error_msg: str = ""):
         self.error_code = str(error_code)
         self.error_msg = error_msg
 
 
 def business_exception_response(exc: BizError) -> JSONResponse:
```
Comment (tc-imba) [inline fastapi_rest_framework/errors.py:55]: business_exception_response seems like a sub type of StandardResponse?
Is it better if we write a class StandardErrorResponse to handle it?

File: example/app.py (hunk 1 above)
Comment (BoYanZh) [reply example/app.py:25]: with that decorator, type hints in python for error code crashes, need to be fixed

File: fastapi_rest_framework/errors.py (hunk 3 above)
Comment (BoYanZh) [reply fastapi_rest_framework/errors.py:55]: We still need to convert that class to json,

**Pull request closed without merging**